src/validators/typings.py

```diff
@@ -36,7 +36,7 @@ def public_keys(self) -> list[HexStr]:
 @dataclass
 class ExitSignatureShards:
     public_keys: list[HexStr]
-    exit_signatures: list[BLSSignature]
+    exit_signatures: list[HexStr]
```

Comment:
I was thinking about making 2 dataclasses - one for raw signatures, another one for encrypted signatures. I ended up with the same dataclass as it was before.